book/intro.md

```diff
@@ -1,41 +1,41 @@
 
-You can find us on:
-- Our websites: [Government Analysis Function](https://www.gov.uk/government/organisations/government-analysis-function) and [Government Statistical Service](https://gss.civilservice.gov.uk/)
-- Email: [analysis.function@ons.gov.uk](mailto:analysis.function@ons.gov.uk) and [gsshelp@statistics.gov.uk](mailto:gsshelp@statistics.gov.uk)
-- Slack: [Government Data Science](https://govdatascience.slack.com) and [GSS](https://gov-stats-service.slack.com)
-- Twitter: [Government Analysis Function](https://twitter.com/gov_analysis),  [GSS Good Practice Team](https://twitter.com/gssgoodpractice) and [UK GSS](https://twitter.com/ukgss)
+The good practices outlined in the book are general to many applications of programming, so may also be relevant for those outside of government.
 
-## Citing the book
+The [RAP learning pathway](https://learninghub.ons.gov.uk/totara/program/view.php?id=6) provides training for many of these good practices. This book can be used to guide your learning and as a reference when applying these practices in your work. Each chapter describes the risks that each practice may help to address. As recommended by [the Aqua book](https://www.gov.uk/government/publications/the-aqua-book-guidance-on-producing-quality-analysis-for-government), you should strive to apply the most appropriate quality assurance practices given the risks associated with your work.
 
-The following structure can be used to reference the current version of the book:
+The principles in this book are language agnostic. The book does not aim to form a comprehensive learning resource and you may often need to study further resources to implement these practices. That said, examples and useful references are provided for **Python** and **R**, as open source languages that are commonly applied across government.
 
-> UK Government Analytical Community. (2020). Quality assurance of code for analysis and research (version 2022.4). Office for National Statistics, Quality and Improvement division: https://best-practice-and-impact.github.io/qa-of-code-guidance/
+## About us
 
-## How to get the most out of the book
+The Analysis Standards and Pipelines hub supports government analysis by providing guidance, consultancy and training.
 
-This guidance is tailored to government analysts who would like to quality assure their code and increase the reproducibility of their analyses. We have tried to interpret the requirements of pieces of guidance (the [principles](/principles.md)) into actions and deliverables for analytical programming.
+You can find us on:
+- Our websites: [Government Analysis Function](https://www.gov.uk/government/organisations/government-analysis-function)
+- Email: [Analysis Standards and Pipelines hub](mailto:ASAP@ons.gov.uk)
+- Slack: [Government Data Science](https://govdatascience.slack.com) and [AF](https://gov-stats-service.slack.com)
```

Comment:
Looks like the AF slack has been suspended.

Reply:
Yep that was recently shut down